src/templates/type/moment-select.html

```diff
@@ -7,7 +7,7 @@
                         class="select"
                         ng-model="hours"
                         ng-disabled="disable"
-                        ng-options="hour as hour for hour in hourChoices">
+                        ng-options="hour as (trailingHours + hour).slice(-(trailingHours.length)) for hour in hourChoices">
```

Comment:
It's very hard to read. A filter with a name describes what it does and cuts down on template code. Also it becomes usable elsewhere.